ui/apps/platform/src/Components/CompoundSearchFilter/components/ConditionDate.tsx

```diff
@@ -16,6 +16,10 @@ export type ConditionDateProps = {
     onSearch: (value: ConditionDate) => void;
 };
 
+function dateFormat(date: Date): string {
+    return format(date, 'MM/DD/YYYY');
+}
```

Comment:
If I understand correctly from your comment, this is temporary solution, therefore better to encapsulate where it is needed, but otherwise we would import the helper function from dateUtils.ts file?

Reply:
Yes, temporary, but I'm inclined to change this to use a general `YYYY-MM-DD` date format for clarity instead, and convert between that and the `MM/DD/YYYY` format the backend expects. 

I am concerned that attempting to detect the format the user is using and do the conversion ourselves will lead to edge cases or difficult to test bugs for different locales that I would rather avoid. Since we are using ISO 8601 elsewhere in the UI, I don't think this is too inconsistent, and the user does still have the option of using the date picker component instead of typing the date in manually.